services/staging/rs_server_staging/main.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2024 CS Group
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""rs server staging main module."""
+# pylint: disable=E0401
+
+import os
+import uuid
+
+from fastapi import FastAPI, HTTPException, Path
+from pydantic import BaseModel
+from pygeoapi.api import API
+from pygeoapi.config import get_config
+from starlette.exceptions import HTTPException as StarletteHTTPException
+from starlette.middleware.cors import CORSMiddleware
+from starlette.requests import Request
+from starlette.responses import JSONResponse
+from starlette.status import HTTP_200_OK, HTTP_404_NOT_FOUND
+from tinydb import Query, TinyDB
+
+from .processors import processors
```

Comment:
It's best to use absolute imports i.e. `from rs_server_staging.processors import processors`

Reply:
Yes